src/api/Shrooms.DataLayer/Migrations/DataInitializer/PermissionInitializer.cs

```diff
@@ -79,7 +80,8 @@ internal static void CreatePermissions(ShroomsDbContext context)
                 .AddAdminPermission(permissionName: AdministrationPermissions.Monitor, withRoleNames: new[] { Roles.Administration }, module: Modules.Monitor)
                 .AddAdminPermission(permissionName: AdministrationPermissions.KudosShop, withRoleNames: Roles.Administration)
                 .AddAdminPermission(permissionName: AdministrationPermissions.Job, withRoleNames: Roles.Administration)
-                .AddAdminPermission(permissionName: AdministrationPermissions.Lottery, withRoleNames: new[] { Roles.LotteryAdmin, Roles.Administration });
+                .AddAdminPermission(permissionName: AdministrationPermissions.Lottery, withRoleNames: new[] { Roles.LotteryAdmin, Roles.Administration })
+                .AddAdminPermission(permissionName: AdministrationPermissions.Blacklist, withRoleNames: new[] { Roles.Admin });
```

Comment:
What is the reason that in other places it's called `Administration` and in here it's `Admin`?